src/cmd/flash.rs

```diff
@@ -108,7 +108,7 @@ pub struct Args {
     #[argh(switch)]
     use_local_image: bool,
 
-    /// flash recovery image (default: flash test image)
+    /// flash recovery image (default: flash test image).
```

Comment:
Do we need a period at the end of the sentence? it looks like no period is better for consistency with other items...

Reply:
Indeed.